### ISSUE TITLE
[yb-platform] parsing of df output is fragile and may fail in case of "safe" error in df

### ISSUE THREAD

**Issue by tylarb** [yugabyte/yugabyte-db#7402]
ISSUE: noticed an error like this:
```
26881 Traceback (most recent call last):
26882   File "bin/cluster_health.py", line 868, in <module>
26883     main()
26884   File "bin/cluster_health.py", line 836, in main
26885     entries = coordinator.run()
26886   File "bin/cluster_health.py", line 709, in run
26887     check.entry = check.result.get() if check.result else None
26888   File "/usr/lib64/python2.7/multiprocessing/pool.py", line 554, in get
26889     raise self._value
26890 ValueError: invalid literal for int() with base 10: 'handl'
```

From df: 

```
df: ‘/NAS/.snapshot/REPL-MTREE-AUTO’: Stale file handle 
Filesystem Size Used Avail Use% Mounted on
devtmpfs 32G 4.0K 32G 1% /dev
tmpfs 32G 224K 32G 1% /dev/shm
tmpfs 32G 76M 32G 1% /run
tmpfs 32G 0 32G 0% /sys/fs/cgroup
/dev/mapper/rootvg-rootlv 92G 8.5G 84G 10% /
/dev/sda1 2.0G 148M 1.9G 8% /boot
/yb-data 300G 25G 276G 9% /yb-data
nas.com:/data/col1/CHL_Yugabyte_PROD 886T 484T 402T 55% /NAS
nas.com:/data/col1/CHL_Yugabyte_PROD/.snapshot 886T 484T 402T 55% /NAS/.snapshot
tmpfs 6.3G 0 6.3G 0% /run/user/0

$ echo $?
1
```

The parsing in health check specifically does some output ranging, and alwasy parses the 4th column as int:

```
248     def get_disk_utilization(self):
249         remote_cmd = 'df -h'
250         return self._remote_check_output(remote_cmd)
251 
252     def check_disk_utilization(self):
253         logging.info("Checking disk utilization on node {}".format(self.node))
254         e = self._new_entry("Disk utilization")
255         found_error = False
256         output = self.get_disk_utilization()
257         msgs = []
258         if has_errors(output):
259             return e.fill_and_return_entry([output], True)
260 
261         # Do not process the headers.
262         lines = output.split('\n')
263         msgs.append(lines[0])
264         for line in lines[1:]: <<<
265             msgs.append(line)
266             if not line:
267                 continue
268             percentage = line.split()[4][:-1] <<<
269             if int(percentage) > DISK_UTILIZATION_THRESHOLD_PCT:
270                 found_error = True
271         return e.fill_and_return_entry(msgs, found_error)
```

So, because the first line isn't the header in df output, instead we try to cast as int the value `Use%` or `handle`

Ideas...
- Only parse disks we care about, ignore all other output? 
- this error caused df to return err code 1 - so maybe df should have failed? (even though it wasn't on the disk we care about in YB)

**Comment by SergeyPotachev** [comment]
The issue could be partly resolved if #5246 is implemented.

**Comment by VijiYB** [comment]
@SergeyPotachev verified on 2.4 portal
